### DISCUSSION TITLE
The thing is

### DISCUSSION THREAD

**Discussion by TNTguy** [LULU1422-2/RedPillEnemySpawn#6] [General]
In the desc of the mod, You mentioned How reports of players appearing to find obunga under the company building, This is false. As i was the host of that lobby and was simply trolling with a mod that would let me spawn him in. it was simple a little troll and little laugh, i can even show you the mod i used.

**Comment by otherotherjosh** [comment]
lmao thanks for letting me know! I'll fix the description with the next update. What mod were you using?

**Comment by LULU1422-2** [comment]
ahahaha i had a feeling it was fake. in early dev tests of this mod when it would kill a player it would still think it would be alive therefore being stuck at the players last known location and not pick another player

https://github.com/otherotherjosh/RedPillEnemySpawn/assets/93459379/04817869-29ce-4693-ba57-3314baf57fba

notice how he doesnt kill me after it kills @otherotherjosh despite having a speed double the players RUN speed

**Comment by TNTguy** [comment]
im glad i could debunk it since i never even planned to spread misinformation around
